kernel/nativelambda.mli

```diff
@@ -53,7 +53,7 @@ and lam_branches =
 and fix_decl =  Name.t Context.binder_annot array * lambda array * lambda array
 
 type evars =
-    { evars_val : existential -> constr option;
+    { evars_val : constr evar_handler;
```

Comment:
Should we merge evar_handler and this type?

Reply:
Probably, I've been willing to introduce meta reduction in the kernel for ages.

Reply:
(Not for this PR yet because if the kernel starts observing metas it might change the behaviour.)